VAMobile/src/api/claimsAndAppeals/getEFolderDocuments.tsx

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { ClaimEFolderData, ClaimEFolderDocuments } from 'api/types'
+import { get } from 'store/api'
+
+import { claimsAndAppealsKeys } from './queryKeys'
+
+/**
+ * Fetch user E Folder Documents
+ */
+const getEfolderDocuments = async (): Promise<Array<ClaimEFolderDocuments> | undefined> => {
+  const response = await get<ClaimEFolderData>(`/v0/efolder/documents`, {})
+  return response?.data
+}
+
+/**
+ * Returns a query for user E Folder Documents
+ */
+export const useEFolderDocuments = (options?: { enabled?: boolean }) => {
+  return useQuery({
```

Comment:
Good call out, I asked BE if any services would affect this or not

Reply:
Okay cool, I'll approve to keep things moving. Depending on their response you can update the PR if appropriate

Reply:
technically yes it would be tied to the claims access, but that is covered in the list call prior to the individual claim so nothing needs to be added here

Reply:
BE got to me and it is covered under claims which is covered under our list call api and doesn't need to be added here